GitHubCard/index.js

```diff
@@ -24,7 +32,19 @@
           user, and adding that card to the DOM.
 */
 
-const followersArray = [];
+const followersArray = ['tetondan', 'dustinmyers', 'justsml', 'luishrd', 'bigknell'];
```

Comment:
I think the project has 3 parts.  They need you to make a card for yourself, cards for the 5 teachers, and then also 5 people you're following.

Reply:
Actually, I think I read that wrong.  The teachers are the people you're "following"